dotnet/samples/Concepts/ChatCompletion/AzureAIInference_ChatCompletion.cs

```diff
@@ -15,9 +16,13 @@ public async Task ServicePromptAsync()
     {
         Console.WriteLine("======== Azure AI Inference - Chat Completion ========");
 
-        var chatService = new AzureAIInferenceChatCompletionService(
-            endpoint: new Uri(TestConfiguration.AzureAIInference.Endpoint),
-            apiKey: TestConfiguration.AzureAIInference.ApiKey);
+        Assert.NotNull(TestConfiguration.AzureAIInference.ApiKey);
+
+        var chatService = new ChatCompletionsClient(
+                endpoint: new Uri(TestConfiguration.AzureAIInference.Endpoint),
+                credential: new Azure.AzureKeyCredential(TestConfiguration.AzureAIInference.ApiKey))
```

Comment:
Should we use api key instead of credential for samples? Same below.

Reply:
There's no option for passing the `apiKey`, it needs to be a class, in this case an `AzureKeyCredential`.